coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -296,9 +296,10 @@ export function getOrganisationPerformer(fhirFirstMedicationDispense: fhir.Medic
 }
 
 export function getMedicationDispenseContained(
-  fhirFirstMedicationDispense: fhir.MedicationDispense
+  fhirMedicationDispense: fhir.MedicationDispense
 ): fhir.MedicationRequest | undefined {
-  return fhirFirstMedicationDispense.contained.find(p => p.resourceType === "MedicationRequest")
+  const authorizingPrescriptionReference = fhirMedicationDispense.authorizingPrescription[0].reference.substring(1)
```

Comment:
change `.substring()` to more readable function

Reply:
done